pint_server/models.py

```diff
@@ -76,7 +76,7 @@ class ProviderServerBase(PintBase):
     # we decided to support other vendors, we'll need to update this
     # column type accordingly.
     ip = Column(postgresql.INET, primary_key=True)
-    region = Column(String(100), primary_key=True)
+    region = Column(String(100))
```

Comment:
As per my comment in the main conversation, you will need to re-work this patch per the https://github.com/SUSE-Enceladus/public-cloud-info-service#how-to-upgrade-schema workflow to be able to land a schema change.

Reply:
Yeah, this change needs to be done as part of schema migration.